packages/vkui/src/lib/floating/customResizeObserver.ts

```diff
@@ -0,0 +1,97 @@
+const defaultIframeStyles = {
+  position: 'absolute',
```

Comment:
```suggestion
  position: 'absolute',
  zIndex: -1,
```

возможно стоит добавить ещё и `zIndex: -1`

Reply:
Думаю, что не помешает. ⚡ https://github.com/VKCOM/VKUI/pull/6572/commits/faa9fb765b27b470908f72b4bd51fe878ac29f70